root-html-file/index.html

```diff
@@ -27,40 +28,36 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.0.0/extras/named-register.min.js"></script>
   </head>
   <body>
+    <single-spa-router style="display: none">
```

Comment:
I changed the documentation for this recently within single-spa-layout - the older examples had `style="display: none"` but the newer examples do not. See related https://github.com/single-spa/single-spa.js.org/pull/270.

Starting with [single-spa-layout@1.0.0-beta.2](https://github.com/single-spa/single-spa-layout/releases/tag/v1.0.0-beta.2), it's now possible to use a `<template>` element instead of `display: none`.

Could you make the following changes to this?

1.
```suggestion
    <single-spa-router>
```

2. Wrap in a `<template>` element:
```html
<template>
  <single-spa-router>
  </single-spa-router>
</template>
```

3. Move the `<template>` to be in `<head>` instead of `<body>`

Reply:
Hi, Joel!
Thanks for the feedback, did not know that the docs have changed! :) Should read the news more often... :)

I am pretty sure I am following the steps you suggest, but I'm getting this error:
```
Unhandled Promise rejection: Invalid routesConfig: received null but expected a plain object ; Zone: <root> ; Task: Promise.then ; Value: Error: Invalid routesConfig: received null but expected a plain object
```

Reply:
Ah so you you should add an `id` to the template and then change your `document.querySelector`.

More info at https://single-spa.js.org/docs/layout-overview#basic-usage